networkdb/networkdb.go

```diff
@@ -200,6 +200,17 @@ func (nDB *NetworkDB) Close() {
 	}
 }
 
+// Peers returns the gossip peers for a given network.
+func (nDB *NetworkDB) Peers(nid string) []string {
+	nDB.RLock()
+	defer nDB.RUnlock()
+	peers := make([]string, 0, len(nDB.networkNodes[nid]))
+	for _, nodeName := range nDB.networkNodes[nid] {
+		peers = append(peers, nDB.nodes[nodeName].Addr.String())
```

Comment:
I think displaying the node name in the gossip cluster along with the IP has value since we auto generate a portion of the name. But we shouldn't return the entire `Node` which is an implementation detail.